src/OpenAI.tsx

```diff
@@ -171,7 +180,20 @@ const CallOpenAi = async ({
         console.debug(json);
 
         if (json.error) {
-          onError(json.error.message);
+          let errorMessage = json.error.message;
+
+          // Check for common trial/quota-related errors and provide helpful messages
+          if (errorMessage.includes('quota') || errorMessage.includes('billing') || errorMessage.includes('exceeded')) {
+            if (usingTrial) {
+              errorMessage = 'Trial quota exceeded. Please add your own OpenAI API key in Settings to continue. Get one at https://platform.openai.com/account/api-keys';
+            } else {
+              errorMessage = `${errorMessage}\n\nGet help managing your OpenAI quota at https://platform.openai.com/account/usage`;
+            }
+          } else if (errorMessage.includes('invalid') && errorMessage.includes('api')) {
+            errorMessage = `${errorMessage}\n\nCheck your API key in Settings or get a new one at https://platform.openai.com/account/api-keys`;
+          }
+
+          onError(errorMessage);
```

Comment:
```suggestion
          onError(json.error.message);
```
I'm not convinced this is worth it. I don't want to take a dependency on open AI's error message contents. And none of this works if the language is different. So let's leave it out.